packages/cli/lib/supportHyperlinks.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const hasFlag = require('./hasFlag');
+const supportsColor = require('./supportsColor');
+
+//See https://github.com/jamestalmage/supports-hyperlinks (License: https://github.com/jamestalmage/supports-hyperlinks/blob/master/license)
+
+function parseVersion(versionString) {
+  if (/^\d{3,4}$/.test(versionString)) {
+    // Env var doesn't always use dots. example: 4601 => 46.1.0
+    const m = /(\d{1,2})(\d{2})/.exec(versionString);
+    return {
+      major: 0,
+      minor: parseInt(m[1], 10),
+      patch: parseInt(m[2], 10),
+    };
+  }
+
+  const versions = (versionString || '').split('.').map(n => parseInt(n, 10));
+  return {
+    major: versions[0],
+    minor: versions[1],
+    patch: versions[2],
+  };
+}
+
+function supportsHyperlink(stream) {
+  const { env } = process;
+
+  if ('FORCE_HYPERLINK' in env) {
+    return !(
+      env.FORCE_HYPERLINK.length > 0 && parseInt(env.FORCE_HYPERLINK, 10) === 0
+    );
+  }
+
+  if (
+    hasFlag('no-hyperlink') ||
+    hasFlag('no-hyperlinks') ||
+    hasFlag('hyperlink=false') ||
+    hasFlag('hyperlink=never')
+  ) {
+    return false;
+  }
+
+  if (hasFlag('hyperlink=true') || hasFlag('hyperlink=always')) {
+    return true;
+  }
+
+  // If they specify no colors, they probably don't want hyperlinks.
+  if (!supportsColor.createSupportsColor(stream)) {
```

Comment:
I think we should probably remove this check because we're separately checking for whether or not the terminal supports color. We shouldn't be assuming that they don't want hyperlinks just because they don't support color.